extras/EditorSupport/vim/syntax/sli.vim

```diff
@@ -0,0 +1,73 @@
+hi def link sliKeyword     Keyword
+
+syn keyword sliConstant M_INFO M_ERROR M_DEBUG M_WARNING M_STATUS M_FATAL
+hi def link sliConstant    Constant
+
+syn match sliUnit '\smV'
+syn match sliUnit '\spF'
+syn match sliUnit '\spA'
+syn match sliUnit '\sms'
+syn match sliUnit '\ss'
```

Comment:
Looking into this - maybe I can add word boundaries or something to restrict the match.